docker-compose not called but docker compose which is not available

I am trying to implement https://github.com/WolfgangFahl/pymediawikidocker/issues/4

when calling:
```python
  docker.compose.up(detach=True)
```
it works on MacOS with
```bash
docker --version
Docker version 20.10.6, build 370c289

docker-compose --version
docker-compose version 1.29.1, build c34c88b2
```
but not on Ubutu 20.04 LTS
with
```bash
docker --version
Docker version 20.10.7, build f0df350
```
The error is:
```
python_on_whales.utils.DockerException: The docker command executed was `/usr/bin/docker compose up --detach`.
It returned with code 125
The content of stdout can be found above the stacktrace (it wasn't captured).
The content of stderr is 'unknown flag: --detach
See 'docker --help'.

Usage:  docker [OPTIONS] COMMAND

A self-sufficient runtime for containers

Options:
      --config string      Location of client config files (default
                           "/home/wf/.docker")
  -c, --context string     Name of the context to use to connect to the
                           daemon (overrides DOCKER_HOST env var and
                           default context set with "docker context use")
  -D, --debug              Enable debug mode
  -H, --host list          Daemon socket(s) to connect to
  -l, --log-level string   Set the logging level
                           ("debug"|"info"|"warn"|"error"|"fatal")
                           (default "info")
      --tls                Use TLS; implied by --tlsverify
      --tlscacert string   Trust certs signed only by this CA (default
                           "/home/wf/.docker/ca.pem")
      --tlscert string     Path to TLS certificate file (default
                           "/home/wf/.docker/cert.pem")
      --tlskey string      Path to TLS key file (default
                           "/home/wf/.docker/key.pem")
      --tlsverify          Use TLS and verify the remote
  -v, --version            Print version information and quit

Management Commands:
  app*        Docker App (Docker Inc., v0.9.1-beta3)
  builder     Manage builds
  buildx*     Build with BuildKit (Docker Inc., v0.5.1-docker)
  config      Manage Docker configs
  container   Manage containers
  context     Manage contexts
  image       Manage images
  manifest    Manage Docker image manifests and manifest lists
  network     Manage networks
  node        Manage Swarm nodes
  plugin      Manage plugins
  scan*       Docker Scan (Docker Inc., v0.8.0)
  secret      Manage Docker secrets
  service     Manage services
  stack       Manage Docker stacks
  swarm       Manage Swarm
  system      Manage Docker
  trust       Manage trust on Docker images
  volume      Manage volumes

Commands:
  attach      Attach local standard input, output, and error streams to a running container
  build       Build an image from a Dockerfile
  commit      Create a new image from a container's changes
  cp          Copy files/folders between a container and the local filesystem
  create      Create a new container
  diff        Inspect changes to files or directories on a container's filesystem
  events      Get real time events from the server
  exec        Run a command in a running container
  export      Export a container's filesystem as a tar archive
  history     Show the history of an image
  images      List images
  import      Import the contents from a tarball to create a filesystem image
  info        Display system-wide information
  inspect     Return low-level information on Docker objects
  kill        Kill one or more running containers
  load        Load an image from a tar archive or STDIN
  login       Log in to a Docker registry
  logout      Log out from a Docker registry
  logs        Fetch the logs of a container
  pause       Pause all processes within one or more containers
  port        List port mappings or a specific mapping for the container
  ps          List containers
  pull        Pull an image or a repository from a registry
  push        Push an image or a repository to a registry
  rename      Rename a container
  restart     Restart one or more containers
  rm          Remove one or more containers
  rmi         Remove one or more images
  run         Run a command in a new container
  save        Save one or more images to a tar archive (streamed to STDOUT by default)
  search      Search the Docker Hub for images
  start       Start one or more stopped containers
  stats       Display a live stream of container(s) resource usage statistics
  stop        Stop one or more running containers
  tag         Create a tag TARGET_IMAGE that refers to SOURCE_IMAGE
  top         Display the running processes of a container
  unpause     Unpause all processes within one or more containers
  update      Update configuration of one or more containers
  version     Show the Docker version information
  wait        Block until one or more containers stop, then print their exit codes

Run 'docker COMMAND --help' for more information on a command.

[1mTo get more help with docker, check out our guides at https://docs.docker.com/go/guides/[0m
```

It looks like
```bash
docker-compose
```
should be called instead of 
```bash
docker compose
```

see https://github.com/WolfgangFahl/pymediawikidocker/runs/2868316276?check_suite_focus=true
```
======================================================================
FAIL: testComposePluginInstalled (tests.test_install.TestInstall)
make sure the docker compose command is available
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/pymediawikidocker/pymediawikidocker/tests/test_install.py", line 27, in testComposePluginInstalled
    self.assertTrue(docker.compose.is_installed())
AssertionError: False is not true
```
_False is not true_  ... indeed

--- a/docs/template/sub-commands/compose.md
+++ b/docs/template/sub-commands/compose.md
@@ -2,7 +2,15 @@
 
 Behind the scenes, 
 the [Go implementation of Docker compose](https://github.com/docker/compose-cli)
-is called, not the [Python implementation](https://github.com/docker/compose).
+is called a.k.a. Compose v2, not the [Python implementation](https://github.com/docker/compose).
+
+You can verify that `docker compose` is installed by running
+```
+docker compose --help
+```
+If that doesn't work, then install the cli plugin.
+The instructions are here: https://github.com/docker/compose-cli#compose-v2-aka-local-docker-compose
+it's just a single binary to download.
 
 The Go implementation of compose is still experimental, so take the appropriate precautions.
 

